account-kit/infra/src/client/viem.ts

```diff
@@ -0,0 +1,33 @@
+import { createBundlerClient } from "viem/account-abstraction";
+import type { AlchemySmartAccountClientConfig } from "./smartAccountClient";
+
+// Yea this isn't great or going to work well, we should just keep using our smart account client, but change the account type to match the viem one
```

Comment:
💬 What about the idea of a transform then from the current client into this new client. Like rust's `Into` traits
Or, is it more worth it for the changing of the account type?